src/components/LoginView.vue

```diff
@@ -94,28 +101,34 @@ export default {
 };
 </script>
 
+<!-- TODO: the login/register view have the exact same style section
+should combine for no repetition -->
+
 <style scoped>
 label {
   display: block;
   margin-top: 10px;
 }
 
 .card-container.card {
-  max-width: 350px !important;
-  padding: 40px 40px;
+  /* TODO: this padding has a direct influence on the size of the page
+  this extends the cards height which increases the size parameters for the picture
+  would like to make it the size of the page if possible*/
```

Comment:
I don't understand, can you message me about this and we'll try to figure it out